lib/Auth/Process/privacyidea.php

```diff
@@ -97,6 +98,15 @@ public function process(&$state)
 		}
 
 		if($piEnabled) {
+			if ($this->serverconfig['tryFirstAuthentication']) {
+				try {
+					if ($this->authenticate($state, "simpleSAMLphp", null, null, null, null)) {
```

Comment:
I understand that in this case not access the previously (in a previous authproc filter) entered password?
So unfortunately we can not use passthru policy in privacyIDEA.

Could we make the password "simpleSAMLphp" configurable and have "simpleSAMLphp" to be the default password.